proto/stakeibc/params.proto

```diff
@@ -28,4 +28,5 @@ message Params {
   uint64 buffer_size = 10;
   uint64 ibc_timeout_blocks = 11;
   uint64 fee_transfer_timeout_nanos = 12;
+  uint64 max_ica_calls_per_epoch = 13;
```

Comment:
I think this PR and your RR check PR will have a merge conflict here, just flagging